packages/fast-usdc-deploy/test/fast-usdc.test.ts

```diff
@@ -114,12 +120,45 @@ test.before('bootstrap', async t => {
     await eventLoopIteration();
   };
 
+  /**
+   * Helper to submit evidence by wallet invoke message from all configured oracles.
+   * @param evidence CCTP evidence
+   * @param invocationIdPrefix Prefix for the invocation ID
+   * @param extraArgs Additional arguments for the submitEvidence call
+   */
+  const invokeAttest = async (
+    evidence: CctpTxEvidence,
```

Comment:
Instead of the `@ts-expect-error` below, would this work?

```suggestion
    evidence: CctpTxEvidence & CopyRecord,
```

or add `& CopyRecord` to the definition of `CctpTxEvidence`

Reply:
That just pushes the problem up I think. Really the problem is the `Passable` type, but that's a fight I don't want to take up now.